src/boot/cic6105.c

```diff
@@ -1,3 +1,85 @@
+#include "global.h"
 #include "cic6105.h"
+#include "fault.h"
 
-u32 B_80008EE0;
+// TODO N64 fault.c functions
+void func_800AE1E0_unknown(s32, s32);
+void func_800AE258_unknown(const char*, ...);
+
+s32 func_80001714(void);
+
+extern u64 cic6105ucodeTextStart[];
+
+OSTask D_800067C0_unknown = {
+    4, 0, rspbootTextStart, 0x3E8, cic6105ucodeTextStart, 0x20, (u64*)gBuildTeam, 8, NULL, 0, NULL, 0, NULL, 0, NULL, 0,
```

Comment:
turns out cic6105ucodeTextStart is actually D_800093F0 (thanks to @Thar0 for pointing that out). So I guess we should probably rename D_800093F0 to cic6105ucodeTextStart (or similar) and use that instead. Not sure if you want to make that change now though

Reply:
actually I guess this should go in a separate file too. That seems like a bigger change so nvm

Reply:
doing nothing then, leaving it to someone else

nice catch Tharo